packages/swapper/src/swappers/zrx/getZrxMinMax/getZrxMinMax.ts

```diff
@@ -19,7 +19,7 @@ export const getZrxMinMax = async (
     tokenId: sellAsset.tokenId
   })
 
-  const minimum = bnOrZero(1).dividedBy(bnOrZero(usdRate)).toString()
+  const minimum = bn(1).dividedBy(bnOrZero(usdRate)).toString()
```

Comment:
This seems like we need to check and error on invalid `usdRate` to avoid divide by zero error which looks like it results in string `Infinity`

Reply:
is it a good idea to assume getUsdRate will always behave that way?

Reply:
I think so, as we maintain it - a change in behavior should be caught in review.

Addressed in https://github.com/shapeshift/lib/pull/639/commits/b294ba1441c3bb2fe451344d6418d83dfd1971ff